### DISCUSSION TITLE
How to scale SumatraPDF correctly with WPF WebBrowser?

### DISCUSSION THREAD

**Discussion by otearaisu** [sumatrapdfreader/sumatrapdf#3924] [Q&A]
Running SumatraPDF with -plugin to WPF WebBrowser.
The display is fine when running it in the original window scale when WPF started.
![wpf1](https://github.com/sumatrapdfreader/sumatrapdf/assets/135924218/80744244-bf61-4459-a61a-f3a1dfa15770)
But when the scale is updated, or using another monitor with a different scale, the display becomes wrong and becomes smaller or larger than the WebBrowser.
![wpf2](https://github.com/sumatrapdfreader/sumatrapdf/assets/135924218/24551e64-4f40-47ea-87ff-1e8572ba7641)

Is there way to fix this so that scale of the SumatraPDF will match the WebBrowser?

**Comment by kjk** [comment]
I'm guessing you would have to resize hwnd embedding Sumatra

**Comment by otearaisu** [comment]
Fixed by setting the dpi awareness in the wpf app
